bin/mail/2016-09-15-latam-hoc-starwars/generate-mailing-lists.rb

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Query all HOC organizers and Code Studio teachers from any of the Latam countries.
+
+# Latin America (latam)
+LATAM_ES_COUNTRIES = [
+  'Mexico',
+  'Guatemala',
+  'El Salvador',
+  'Costa Rica',
+  'Nicaragua',
+  'Belize',
+  'Panama',
+  'Cuba',
+  'Dominican Republic',
+  'Venezuela',
+  'Colombia',
+  'Ecuador',
+  'Peru',
+  'Bolivia',
+  'Paraguay',
+  'Uruguay',
+  'Chile',
+  'Argentina',
+  'Puerto Rico'
+].freeze
+es_country_or_list = LATAM_ES_COUNTRIES.join(' OR ')
+
+LATAM_ES_COUNTRY_CODES = %w(mx gt sv cr ni bz pa cu do ve co ec pe bo py uy cl ar pr).freeze
+es_country_code_or_list = LATAM_ES_COUNTRY_CODES.join(' OR ')
```

Comment:
It seems more readable to define a hash `COUNTRY_CODE_TO_COUNTRY_NAME`.

Then `ES_COUNTRY_OR_LIST = COUNTRY_CODE_TO_COUNTRY_NAME.values.join(' OR ').freeze` and so on.